test/mocks.ts

```diff
@@ -26,4 +26,35 @@ module Mocks {
       return true;
     }
   }
+
+  export class Scale<D, R> extends Plottable.Scale<D, R> {
```

Comment:
However, it doesn't test the subclasses of interest (sometimes a problem with `Time`, since `Date`s with the same value are not strict equal), and adds a potential failure mode because now we also have to prove that there aren't problems in the mock.